src/main.rs

```diff
@@ -0,0 +1,32 @@
+extern crate clap;
+use clap::{Arg, App};
+
+mod readfile;
+mod writefile;
+mod agecalc;
```

Comment:
bad module names.....follow snake case naming convention for module names as well